dev-docs/RFCs/v7.x/post-process-effect-rfc.md

```diff
@@ -0,0 +1,66 @@
+# RFC: Post-processing Effect Rendering
+
+* **Authors**: Jian Huang
+* **Date**: May. 2019
+* **Status**: For Review
+
+## Abstract
+Post-processing effect rendering is part of deck.gl effect rendering pipeline, focusing on screen space pixel manipulation by using ping-pong technique.
+
+## Design
+
+A post-processing effect is created from **PostProcessEffect** class and **Shader module**.
+```js
+import {Convolution} from @luma.gl/Effects;
```

Comment:
No uppercase is allowed in a module's name https://docs.npmjs.com/files/package.json#name

Reply:
Got it, thanks!